automation/test-linux.sh

```diff
@@ -63,48 +69,56 @@ fi
 
 delete_vm(){
   vm_name=$1
-  local template_name=$2
+
   set +e
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc process $2 -n $namespace -o json NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
```

Comment:
You have the logic of ${KUBE_CMD} to define when to use `oc` or `kubectl`. However, I see many `oc` in the script. 

I understand that some commands only exist for `oc` (e.g., `process`), but if the script is dependent on `oc` does it make sense to have the ${KUBE_CMD} logic?

If yes, should/can the parts that use `oc` be skiped?

Reply:
Refer to previous comment